src/main/java/subway/line/Line.java

```diff
@@ -0,0 +1,63 @@
+package subway.line;
+
+import subway.section.Section;
+import subway.station.Station;
+
+import javax.persistence.*;
+import java.util.ArrayList;
+import java.util.List;
+
+@Entity
+public class Line {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @Column(length = 15, nullable = false)
+    private String name;
+
+    @Column(length = 20, nullable = false)
+    private String color;
+
+    @OneToMany(fetch = FetchType.LAZY)
```

Comment:
```suggestion
@OneToMany(mappedBy = "line", fetch = FetchType.LAZY)
```
연관관계 주인을 지정해주지 않으면, 중간 테이블이 생성됩니다 😢 
<img width="592" alt="스크린샷 2023-07-28 오후 4 06 14" src="https://github.com/next-step/atdd-subway-map/assets/17218212/71f7ce5d-bc9e-497d-8b0d-afab1188f80b">